db_test.go

```diff
@@ -707,7 +707,9 @@ func TestWALSegmentSizeOption(t *testing.T) {
 		testutil.Ok(t, app.Commit())
 	}
 
-	files, err := ioutil.ReadDir(filepath.Join(db.Dir(), "wal"))
+	dbDir := db.Dir()
+	db.Close()
```

Comment:
This will close on every iteration, so I'd expect the 2nd iteration to fail.

Reply:
sorry I didn't understand. There is no loop here.

Reply:
Ah, indentation was showing oddly on the for above.

Reply:
yeah, I was just going to comment on that as well. It is all good the tests are passing